test/models/laboratory_test.rb

```diff
@@ -90,4 +90,74 @@ def teardown; end
     assert patient.negative_lab_count.zero?
     assert_empty patient.laboratories
   end
+
+  test 'validates result inclusion' do
+    lab = create(:laboratory)
+
+    lab.result = 'positive'
+    assert lab.valid?
+
+    lab.result = ''
+    assert lab.valid?
+
+    lab.result = nil
+    assert lab.valid?
+
+    lab.result = 'foo'
+    assert_not lab.valid?
+  end
+
+  test 'validates lab_type inclusion' do
+    lab = create(:laboratory)
+
+    lab.lab_type = 'PCR'
```

Comment:
Add check for all the other lab types as well.

Reply:
Addressed in https://github.com/SaraAlert/SaraAlert/pull/582/commits/9106e97af2c69a78aee72135c1c39199e6823f83.